tests/Unit/Core/Grid/Position/GridPositionUpdaterTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * 2007-2018 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace Tests\Unit\Core\Grid\Position;
+
+use PHPUnit\Framework\TestCase;
+use PrestaShop\PrestaShop\Core\Grid\Position\GridPositionUpdater;
+use PrestaShop\PrestaShop\Core\Grid\Position\PositionDefinition;
+use PrestaShop\PrestaShop\Core\Grid\Position\PositionModification;
+use PrestaShop\PrestaShop\Core\Grid\Position\PositionModificationCollection;
+use PrestaShop\PrestaShop\Core\Grid\Position\PositionUpdate;
+use PrestaShop\PrestaShop\Core\Grid\Position\UpdateHandler\PositionUpdateHandlerInterface;
+
+class GridPositionUpdaterTest extends TestCase
+{
+    public function testUpdate()
+    {
+        $positionUpdate = $this->createPositionUpdate();
+        $updateHandler = $this->createUpdateHandlerMock();
+        $gridUpdater = new GridPositionUpdater($updateHandler);
+
+        $gridUpdater->update($positionUpdate);
```

Comment:
you have no assertions in this test, is it normal?

Reply:
I could add a test on the first parameter to check the instance's class